kubeadm/kubeadm.go

```diff
@@ -31,6 +49,7 @@ func New() *Kubeadm {
 	return NewWithCmdRunner(cmd_runner.New())
 }
 
+// TokenCreate execs `kubectl token create` with the appropriate flags added by interpreting the params argument.
```

Comment:
To someone calling this method, what's more important here is what is returned and in what format.

Reply:
Good point, updating to talk about the return value and its format.